src/models/App.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * @flow
+ */
+
+import Immutable from 'immutable';
+import Logger from '../utils/Logger';
+
+import {
+  isDefined,
+  isEmptyArray,
+  isNonEmptyString
+} from '../utils/LangUtils';
+
+import { isValidUuidArray } from '../utils/ValidationUtils';
+
+const LOG = new Logger('App');
+
+/**
+ * @class App
+ * @memberof lattice
+ */
+export default class App {
```

Comment:
You're missing the "id" field.

https://github.com/openlattice/api/blob/develop/src/main/java/com/openlattice/apps/App.java#L24